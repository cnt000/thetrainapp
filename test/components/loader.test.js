import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Loader from '../../src/components/loader'

describe('Loader', () => {
  it('renders without crashing isLoading', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Loader isLoading />, div)
  })

  it('renders without crashing isLoading False', () => {
    const div = document.createElement('div')
    div.innerHTML = 'This is and error'
    ReactDOM.render(<Loader isLoading={false} />, div)
  })
})
