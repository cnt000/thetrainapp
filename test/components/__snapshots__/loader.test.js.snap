// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader has a valid snapshot 1`] = `
<div
  className="sc-bdVaJa bsNmEK"
>
  <span
    className="sc-bwzfXH gRRlat"
  >
     Is loading... 
  </span>
</div>
`;
